src/views/Pools/components/PoolsTable/Cells/EarningsCell.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Skeleton, Text, useTooltip, HelpIcon, Flex, Box, useModal } from '@pancakeswap/uikit'
+import { Pool } from 'state/types'
+import BigNumber from 'bignumber.js'
+import { PoolCategory } from 'config/constants/types'
+import { BIG_ZERO } from 'utils/bigNumber'
+import { formatNumber, getBalanceNumber, getFullDisplayBalance } from 'utils/formatBalance'
+import Balance from 'components/Balance'
+import { useCakeVault, useBusdPriceFromToken } from 'state/hooks'
+import { useTranslation } from 'contexts/Localization'
+import { convertSharesToCake } from 'views/Pools/helpers'
+import BaseCell, { CellContent } from './BaseCell'
+import CollectModal from '../../PoolCard/Modals/CollectModal'
+
+interface EarningsCellProps {
+  pool: Pool
+  account: string
+  userDataLoaded: boolean
+  isAutoVault?: boolean
+}
+
+const StyledCell = styled(BaseCell)`
+  flex: 4.5;
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex: 1 0 120px;
+  }
+`
+
+const HelpIconWrapper = styled.div`
+  align-self: center;
+`
+
+const EarningsCell: React.FC<EarningsCellProps> = ({ pool, account, userDataLoaded, isAutoVault }) => {
+  const { t } = useTranslation()
+  const { sousId, earningToken, poolCategory, userData } = pool
+  const isManualCakePool = sousId === 0
+
+  const earnings = userData?.pendingReward ? new BigNumber(userData.pendingReward) : BIG_ZERO
+  const earningTokenPrice = useBusdPriceFromToken(earningToken.symbol)
+  const earningTokenPriceAsNumber = earningTokenPrice && earningTokenPrice.toNumber()
+  // These will be reassigned later if its Auto CAKE vault
+  let earningTokenBalance = getBalanceNumber(earnings, earningToken.decimals)
+  let earningTokenDollarBalance = getBalanceNumber(
+    earnings.multipliedBy(earningTokenPriceAsNumber),
+    earningToken.decimals,
+  )
+  let hasEarnings = account && earnings.gt(0)
+  const fullBalance = getFullDisplayBalance(earnings, earningToken.decimals)
+  const formattedBalance = formatNumber(earningTokenBalance, 3, 3)
+  const earningsDollarValue = formatNumber(earningTokenDollarBalance)
+  const isBnbPool = poolCategory === PoolCategory.BINANCE
+
+  // Auto CAKE vault calculations
+  const {
+    userData: { cakeAtLastUserAction, userShares, lastUserActionTime },
+    pricePerFullShare,
+  } = useCakeVault()
+  const hasAutoEarnings =
+    account && cakeAtLastUserAction && cakeAtLastUserAction.gt(0) && userShares && userShares.gt(0)
+  const { cakeAsBigNumber } = convertSharesToCake(userShares, pricePerFullShare)
+  const cakeProfit = cakeAsBigNumber.minus(cakeAtLastUserAction)
+  const cakeToDisplay = cakeProfit.gte(0) ? getBalanceNumber(cakeProfit, 18) : 0
+
+  const dollarValueOfCake = cakeProfit.times(earningTokenPriceAsNumber)
+  const dollarValueToDisplay = dollarValueOfCake.gte(0) ? getBalanceNumber(dollarValueOfCake, 18) : 0
```

Comment:
Is this the same as https://github.com/pancakeswap/pancake-frontend/pull/1258/files/b4bf66aee61da2b82999acd9b55da26ec710d8c0#diff-402e4cb9074d844667b90f43f90e042e9bc0d3fa8bdcd9bae57434dd5010373eR55-R58?